docs/guides/dns-over-https.md

```diff
@@ -121,5 +121,17 @@ Finally, configure Pi-hole to use the local `cloudflared` service as the upstrea
 
 (don't forget to hit Return or click on `Save`)
 
+### Updating cloudflared install package post install
+
+#### AMD64 architecture (most devices)
+
+Download the  latest installer package, then use `apt-get` to upgrade the package along with any dependencies. Proceed to run the binary with the `-v` flag to check it is all working.
```

Comment:
There is a general issue with this page where the installation is done for ARM and AMD64 in different ways (binary vs DEB), and now the update is only done for AMD64. Can you generalize the install and update by referencing https://developers.cloudflare.com/argo-tunnel/downloads/ for the download source, and only show the DEB install method?